sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -4,6 +4,9 @@
 select TIMESTAMP_SECONDS(1230219000),TIMESTAMP_SECONDS(-1230219000),TIMESTAMP_SECONDS(null);
 select TIMESTAMP_MILLISECONDS(1230219000123),TIMESTAMP_MILLISECONDS(-1230219000123),TIMESTAMP_MILLISECONDS(null);
 select TIMESTAMP_MICROSECONDS(1230219000123123),TIMESTAMP_MICROSECONDS(-1230219000123123),TIMESTAMP_MICROSECONDS(null);
+-- overflow exception:
+select TIMESTAMP_SECONDS(1230219000123123),TIMESTAMP_SECONDS(-1230219000123123),TIMESTAMP_SECONDS(null);
```

Comment:
These cases don't need the "null" case. Just one case per SELECT please, since you can only test one exception at a time -- the first one that gets triggered, the other one isn't even evaluated.

Reply:
Ok,correct right now.